index.html

```diff
@@ -32,60 +32,67 @@
-          <h3 id="cat-week">Archer of the Week!</h3>
+        <article class="col span_8_of_12">
+          <h3>THE History of Academy Archery!</h3>
+          <p>Straight to the Point Archery is a non profit archery education center founded in 2005 by Giap Nguyen. Our state of the art educational and training facility incorporates detailed technical video analysis combined with generations of archery experience from certified coaches and instructors.</p>
+  				<p>Straight to the Point Archery is an indoor training facility with 28 shooting (14 - 10 yard lanes and 14 – 20 yard lanes) and 2 private coaching rooms. We also have access to 26 additional 20 yard lanes and an outdoor shooting field. We offer several types of lessons to accommodate everyone from the first time archer to the professional archer. Once set up with proper fitting equipment, each archer will be given the basic safety instruction to start and will be introduced to various shooting techniques.</p>
+  				<p>From Beginner to Elite, archers of all ages and ability learn about this exciting Olympic and international sport. Sign up for a group lesson, private class, schedule a birthday or corporate party or compete at one of our many tournaments. Straight to the Point Archery also offers homeschooling programs, kids club events, YMCA classes, Boy and Girl Scouts achievement programs and several other youth group activities. We are dedicated to teaching valuable life skills to the youth in our community through the wonderful discipline of archery.</p>
+  				<p>Straight to the Point Archery has instructors that are qualified level 1 and level 2 instructors and have met the safety requirements of the USA Archery (USAA). We teach in a positive learning environment. Our instructors also have extensive knowledge of the most advanced archery equipment and all forms of archery from the traditional long bow to the modern compound. Straight to the Point Archery also has specific instructors for recreational archery, competitive archery and bow hunting.</p>
+  				<p>Straight to the Point Archery has an excellent network of contacts with the USA Archery (USAA), Washington State Archery Association (WSAA), Washington State Bow Hunters (WSB), National Field Archery Association (NFAA) and many more national and state archery associations. These partnerships allow you to feel confident in our facility and staff.</p>
+  				<p>Straight to the Point Archery’s staff has decades of experience in the archery industry. We are committed to educating you with any questions that you may have.</p>
+
+        </article>
+
+
+        <article class="col span_4_of_12">
+          <h3 id="archer">Archer of the Week!</h3>
           <figure>
-						<img src="./images/robin.jpg" alt="We're men, men in tights... tight tights!" style="width:100%">
+						<video width="100%" autoplay="true" controls="controls" src="https://media.istockphoto.com/videos/man-shooting-bow-video-id482997698"></video>

```

Comment:
Check the HTML validator, it will tell you that `100%` is not a valid value to set to the width attribute for the `<video>` tag.